package.json

```
@@ -133,7 +133,7 @@
     "pretest:coverage": "npm run build",
     "release": "semantic-release",
     "start": "npm run build && node dist/cli.js",
-    "test": "cross-env TEST=1 nyc --extension .ts mocha --require ts-node/register --reporter spec --timeout 5000 \"test/**/*.test.ts\"",
+    "test": "cross-env TEST=1 nyc --extension .ts mocha --require ts-node/register --reporter spec --timeout 6000 \"test/**/*.test.ts\"",
```

```suggestion
    "test": "cross-env TEST=1 nyc --extension .ts mocha --require ts-node/register --reporter spec --timeout 10000 \"test/**/*.test.ts\"",
```

If you wanna increase timeout, you shouldn't increase it by 1s, but as doubled previous value, so 5s -> 10s. If it's really problem with timeout, 1s probably won't be a fix, better to have some "backup" of timeout :)

---

😄 sure let's have some backup timeout.